STARTUP.md

```diff
@@ -1,19 +1,19 @@
 # Getting Started
 
-Here you'll find some helpful guidance on how to locally build and run this project. The published [Windows Store app](https://www.microsoft.com/en-us/store/p/windows-ui-dev-labs-sample-gallery/9pp1sb5wgnww) is available as an alternative if you prefer to check out the samples without building the codebase locally. 
+Here you'll find some helpful guidance on how to locally build and run this project. 
 
 ## Prerequisites
 
-For local development, you’ll need Visual Studio 2017 and the Windows Software Development Kit (SDK) for Windows 10.  A free copy of Visual Studio 2017 Community Edition is available [here](http://go.microsoft.com/fwlink/?LinkID=280676). The latest Windows SDK is available [here](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk), and archived SDK releases can be found on the [Windows SDK archive site](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive).
-
-To stay on top of the latest updates to Windows and development tools, become a Windows Insider by [joining the Windows Insider Program](https://insider.windows.com/). After joining, Windows Insider SDKs can be found [here](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK). More information on the Insider program can be found in our [Wiki page](https://github.com/Microsoft/WindowsCompositionSamples/wiki) or on the program [join page](https://insider.windows.com/).
+For local development, please visit this [link](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=vs%2Cstable) to set up your development environment before building this project.
 
 ## Cloning and Building
 
 Once you have Visual Studio and the appropriate SDK(s) installed, clone the Windows Composition Samples repo locally. Cloning instructions can be found [here](https://help.github.com/articles/cloning-a-repository/). 
 
 Open the SampleGallery solution file (.sln extension) in Visual Studio to view and manage the project.
 
+Open up the Solution Explorer (Under the "View" menu item in the top left), right click the SampleGalleryPkg and click "Set as Startup Project".You will also need to change the solution platform to x64 instead of Arm64.
```

Comment:
```suggestion
Open up the Solution Explorer (Under the "View" menu item in the top left), right click the SampleGalleryPkg and click "Set as Startup Project". You will also need to change the solution platform to x64 instead of Arm64.
```